docs/developers/setup-eks.md

```diff
@@ -0,0 +1,56 @@
+# Prerequisite
+
+* [Install eksctl CLI](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html)
+* [Install kubectl CLI](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html)
+* [Install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+
+## Set UP EKS Cluster
+
+### EKS Cluster
+
+[Create an EKS Cluster](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) if you don't have
+one.
+
+### IAM Policy
+
+1. Open the [IAM console](https://console.aws.amazon.com/iam/).
+2. In the navigation pane, choose **Policies**.
+3. Choose **Create policy, JSON**.
+4. Enter the following policy:
+
+```json
```

Comment:
Attaching existing policy is actually better than using inline policy. Though I think @mxiamxia failed to create a new managed policy for otel collector? He mentioned it some time last year ...

Reply:
Replaced the inline policy with multi existing policies.